cmd/csi_driver/main.go

```diff
@@ -50,7 +50,7 @@ var (
 	identityPool                   = flag.String("identity-pool", "", "The Identity Pool to authenticate with GCS API.")
 	identityProvider               = flag.String("identity-provider", "", "The Identity Provider to authenticate with GCS API.")
 	enableProfiling                = flag.Bool("enable-profiling", false, "enable the golang pprof at port 6060")
-	enableBucketScanner            = flag.Bool("enable-bucket-scanner", false, "enable the bucket scanner feature")
+	enableGcsfuseProfiles          = flag.Bool("enable-gcsfuse-profiles", false, "enable the gcsfuse profiles feature")
```

Comment:
@uriel-guzman @chrisThePattyEater  We don't need to change the name of enable-bucket-scanner in this PR also. Let's leave `enable-bucket-scanner` as is, and that will be done in a follow up PR. 

Please only add"enable-gcsfuse-profiles-internal" flag, and leave existing flags as is